packages/admin-next/dashboard/src/providers/router-provider/v2.tsx

```diff
@@ -91,6 +91,27 @@ export const v2Routes: RouteObject[] = [
               },
             ],
           },
+          {
+            path: "inventory",
+            handle: {
+              crumb: () => "Inventory",
+            },
+            lazy: () => import("../../v2-routes/inventory"),
+            children: [
+              {
+                path: "",
+                lazy: () => import("../../v2-routes/inventory/list"),
+              },
+              {
+                path: "locations",
+                lazy: () => import("../../v2-routes/inventory/locations"),
+              },
+              {
+                path: "transfers",
+                lazy: () => import("../../v2-routes/inventory/transfers"),
+              },
+            ],
```

Comment:
suggestion: I think we should split these into separate routes instead of nesting them, otherwise the breadcrumbs will end up weird.